src/QuantumSdk/README.md

```diff
@@ -3,7 +3,7 @@
 ## Content ##
 
 
-The NuGet package Microsoft.Quantum.Sdk serves a .NET Core Sdk for developing quantum applications, meaning it acts as [shared SDK Component](https://docs.microsoft.com/en-us/dotnet/core/tools/cli-msbuild-architecture#the-tooling-layers).
+The NuGet package Microsoft.Quantum.Sdk serves a .NET Core Sdk for developing quantum applications, meaning it acts as [shared SDK Component](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/overview).
```

Comment:
I am not 100% confident that this new link is equivalent to the previous one, but it seems to be the best match (from what I could find).

Reply:
```suggestion
The NuGet package Microsoft.Quantum.Sdk serves a .NET Core Sdk for developing quantum applications, meaning it acts as [shared SDK Component](https://docs.microsoft.com/dotnet/core/project-sdk/overview).
```